components/InteriorLeftNavItem.js

```diff
@@ -8,8 +8,8 @@ const propTypes = {
   activeHref: PropTypes.string,
   tabIndex: PropTypes.number,
   onClick: PropTypes.func,
-  label: PropTypes.string.isRequired,
   blankTarget: PropTypes.bool,
+  children: PropTypes.node,
```

Comment:
I'm using ESLint with `airbnb` as the preset. The rule complains if anything in `Component.propTypes` is both A) not marked `.isRequired` and B) doesn't have a deafult set in `defaultProps`.

My `.eslintrc`, in case you want to play with this:
```json
{
  "root": true,
  "parser": "babel-eslint",
  "parserOptions" : {
    "ecmaVersion": 6,
    "sourceType": "module"
  },
  "env": {
    "es6": true,
    "browser": true,
    "node": false
  },
  "ecmaFeatures": {
    "modules": true
  },
  "extends": [
    "airbnb"
  ],
  "globals": {
  "_contextRoot": true
  },
  "rules" : {
    "max-len": [2, 130, 4],
    "no-plusplus:": 0,
    "no-underscore-dangle": 0,
    "no-param-reassign": ["error", { "props": false }],
    "react/jsx-filename-extension": [1, { "extensions": [".js"] }]
  }
}
```